docs/UserGuide.md

```diff
@@ -296,45 +297,50 @@ Examples:
 * `find appointment bob`
 * `find appointment sunday`
 
-#### Searching clients: `find client`
+#### 3.8.3 Searching clients: `find client`
 
 Finds appointments that matches the keywords and properties whose clients matches the same keywords. Both are done at the same time. 
```

Comment:
think the 'Both done at the same time' is unnecessary? If anything, maybe 'simultaneously' will be better

Reply:
@w-yuchen to update this section